mechanics/MATURITY-LEVELS.md

```diff
@@ -63,7 +63,7 @@ Additional requirements beyond "initiating":
     the release artifacts. Commercial products with production usage count
     towards this target. This can be documented in something
     [like an ADOPTERS.md file](https://www.google.com/url?q=https://github.com/cncf/toc/blob/master/process/graduation_criteria.adoc&sa=D&ust=1597952611892000&usg=AFQjCNFymwghRnNGVqbD0O_01TsfEeto5w).
-- Unit test automation meeting the Knative standards with respect to coverage
+- Unit test automation meeting the Knative standards with respect to coverage for usable components (50% + using [codecov](https://app.codecov.io/gh/knative/serving))
```

Comment:
Can we just say "Test automation" here (and below)? Our coverage in `knative/func` comes from both unit and integration tests.

Reply:
I think integration test automation goes above and beyond.

I think "usable" should probably have unit test automation and coverage analysis. I'm okay with a separate line item saying "overall test coverage exceeds 50%" (which is not a high bar, but does require some effort in writing tests and thinking about coverage).

If we separate it into two different lines, then the line for stable could simply say "test coverage greater than 70%"